packages/gatsby-plugin-vtex-nginx/src/gatsby-node.ts

```diff
@@ -83,7 +83,13 @@ const Node: GatsbyNode = {
 
     writeFileSync(
       join(program.directory, 'public', VTEX_NGINX_CONF_FILENAME),
-      generateNginxConfiguration(rewrites, redirects, headers, files, options)
+      generateNginxConfiguration({
+        rewrites,
+        redirects,
+        headersMap: headers,
+        files,
+        options,
+      })
```

Comment:
We encourage to use objects for functions with 4 or more parameters (vscode can do the refactor for us 😁 )